src/browser/core_state.js

```diff
@@ -124,6 +124,8 @@ function appByUuid(uuid) {
     return coreState.apps.find(app => uuid === app.uuid);
 }
 
+const getAppByUuid = appByUuid;
```

Comment:
So I spent an embarrassing amount of time looking for how to get an app by uuid. All the other getters start with `get` so this is just for consistency: 

````js
appByUuid,
// ...
// ...
getAppById,
getAppByWin,
getAppObj,
getAppObjByUuid,
````